use-timescale/compression/compression-design.md

```diff
@@ -0,0 +1,128 @@
+is now smaller on disk than the previous format.
+
+This format arranges the data so that similar data, such as timestamps, device
+IDs, or temperature readings, is stored contiguously. This means that you can
+then use type-specific compression algorithms to compress the data further, and
+each array is separately compressed. For more information about the compression
+methods used, see the [compression methods section][compression-methods].
+
+When the data is in array format, you can perform queries that require a subset
+of the columns very quickly. For example, if you have a query like this one, that
+asks for the average temperature over the past day:
+
+<CodeBlock canCopy={false} showLineNumbers={false} children={`
+SELECT time_bucket(‘1 minute’, timestamp) as minute
+ AVG(temperature)
+FROM table
+WHERE timestamp > now() - interval ‘1 day’
+ORDER BY minute DESC
+GROUP BY minute;
+`} />
+
+The query engine can fetch and decompress only the timestamp and temperature
+columns to efficiently compute and return these results.
+
+Finally, TimescaleDB uses non-inline disk pages to store the compressed arrays.
+This means that the in-row data points to a secondary disk page that stores the
+compressed array, and the actual row in the main table becomes very small,
+because it is now just pointers to the data. When data stored like this is
+queried, only the compressed arrays for the required columns are read from disk,
+further improving performance by reducing disk reads and writes.
+
+## Querying compressed data
+
+In the previous example, the database has no way of knowing which rows need to
+be fetched and decompressed to resolve a query. For example, the database can't
+easily determine which rows contain data from the past day, as the timestamp
+itself is in a compressed column. You don't want to have to decompress all the
+data in a chunk, or even an entire hypertable, to determine which rows are
+required.
+
+TimescaleDB automatically includes more information in the row and includes
+additional groupings to improve query performance. When you compress a
+hypertable, either manually or through a compression policy, it can help to specify
+an `ORDER BY` column.
+
+`ORDER BY` columns specify how the rows that are part of a compressed batch are
+ordered. For most time-series workloads, this is by timestamp, so if you don't
+specify an `ORDER BY` column, TimescaleDB defaults to using the time column. You
+can also specify additional dimensions, such as location.
+
+For each `ORDER BY` column, TimescaleDB automatically creates additional columns
+that store the minimum and maximum value of that column. This way, the query
+planner can look at the range of timestamps in the compressed column, without
+having to do any decompression, and determine whether the row could possibly
+match the query.
+
+When you compress your hypertable, you can also choose to specify a `SEGMENT BY`
```

Comment:
There is no `SEGMENT BY` in SQL, but this format suggest there is, which is a little confusing.
```suggestion
When you compress your hypertable, you can also choose to specify a segment-by
```